org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/corrections/proposals/FixCorrectionProposal.java

```diff
@@ -37,7 +37,11 @@
 	private CompilationUnit fCompilationUnit;
 
 	public FixCorrectionProposal(IProposableFix fix, ICleanUpCore cleanUp, int relevance, IInvocationContext context) {
-		super(fix.getDisplayString(), CodeActionKind.QuickFix, context.getCompilationUnit(), null, relevance);
+		this(fix, cleanUp, relevance, context, CodeActionKind.QuickFix);
```

Comment:
Without the new constructor, how did it pass the code action kind parameter before?

Reply:
The code action kind is inherited, and was hardcoded as quickfix before.